index.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+// See https://github.com/jshint/jshint/issues/1747 for context
+/* global -Promise */
+var Promise = require('es6-promise').Promise;
+var adb = require('adbkit');
+var portfinder = require('portfinder');
+
+var REMOTE_DEBUGGER_SOCKET = 'localfilesystem:/data/local/debugger-socket';
+
+module.exports = forwardPorts;
+
+function forwardPorts(options) {
+  options = options || {};
+  var devices = options.devices;
+
+  var client = adb.createClient();
```

Comment:
I'd suggest renaming this variable to `adbClient` or similar. It confused me with *our* `firefoxClient` and I was about to write a comment saying that you should not tie forwards to clients (firefox clients) blablabla